ReactAndroid/src/main/jni/react/JSCExecutor.cpp

```diff
@@ -277,7 +277,8 @@ void JSCExecutor::flush() {
 
   if (!ensureBatchedBridgeObject()) {
     throwJSExecutionException(
-        "Couldn't get the native call queue: bridge configuration isn't available. This shouldn't be possible. Congratulations.");
+        "Couldn't get the native call queue: bridge configuration isn't available. You may need to use \"react-native start\" manually, "
```

Comment:
It should be `npm start` instead. But since there are all these different ways to start the server, We should just say `You might need to start the React Native packager.`